src/fragments/lib/logging/ios/hub_events/hub_events.mdx

```diff
@@ -0,0 +1,50 @@
+<BlockSwitcher>
+
+<Block name="Listener">
+
+```swift
+import Amplify
+```
+
+```swift
+// Assumes `unsubscribeToken` is declared as an instance variable in your view
+unsubscribeToken = Amplify.Hub.listen(to: .logging) { payload in
+	switch payload.eventName {
```

Comment:
nit: This file seems to be using tabs instead of spaces for indentation. It might look weird when rendered.

Reply:
tab indentation removed.